server/index.d.ts

```diff
@@ -1,32 +1,25 @@
-import BaseServer, { Logger, Reporter, BaseServerOptions } from '../base-server'
+import BaseServer, { Logger, BaseServerOptions } from '../base-server'
 
-export type ServerOptions = BaseServerOptions & {
+export type LoggerOptions = {
   /**
-   * Custom reporter for process/errors. You should use it only for test purposes
-   * or unavoidable hacks.
-   *
-   * ```js
-   * new Server({
-   *   …,
-   *   reporter: (name, details) => {
-   *     console.log('Event:', name)
-   *     console.log('Details:', JSON.stringify(details))
-   *   }
-   * })
-   * ```
+   * Logger message format.
    */
-  reporter?: Reporter
+  type?: 'human' | 'json'
 
   /**
-   * Stream to be used by reporter to write log.
+   * Stream to be used by logger to write log.
+   * todo Note about flushSync
```

Comment:
Streams should have `flushSync` method to guarantee final log writes. Do we need to document this?

Reply:
Yeap. And we may want to define it in types.